mesonbuild/dependencies/python.py

```diff
@@ -43,6 +43,7 @@ class PythonIntrospectionDict(TypedDict):
         paths: T.Dict[str, str]
         platform: str
         suffix: str
+        limited_api_suffix: str
```

Comment:
The documentation calls this `limited_api`, here it is `limited_api_suffix`. They should be aligned.

Reply:
This is part of the introspection data gathered from the interpreter the extension is being built for, it does not indicate whether the extension is built for the limited API or not. I think it is fine as it is.